var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -46,13 +46,13 @@ class Octopus(Package):
     # FEAST, Libfm, PFFT, ISF, PNFFT
 
     def install(self, spec, prefix):
+        lapack = spec['lapack'].lapack_libs
+        blas = spec['blas'].blas_libs
         args = []
         args.extend([
             '--prefix=%s' % prefix,
-            '--with-blas=%s' % to_link_flags(
-                spec['blas'].blas_shared_lib),
-            '--with-lapack=%s' % to_link_flags(
-                spec['lapack'].lapack_shared_lib),
+            '--with-blas=%s' % ' '.join(blas.ld_flags),
```

Comment:
@davydden Yes, I missed this one.. Can I delete the commented code in this package, or is it there for a reason ?